content/components/form-control.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Form control
-description: Renders a labelled input and, optionally, associated validation text and/or hint text.
+description: Form control renders a labelled input and, optionally, associated validation text and/or hint text.
```

Comment:
Nitpick: I prefer “displays” over “renders” for generic pattern descriptions. “Renders” feels a bit like technical jargon.

```suggestion
description: Form control displays a labelled input and, optionally, associated validation text and/or hint text.
```